src/test/java/core/basesyntax/RegistrationServiceTest.java

```diff
@@ -0,0 +1,81 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.UserRegisterException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceTest {
+
+    private RegistrationServiceImpl registrationService;
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_ok() {
+        User newUser = new User();
+        newUser.setLogin("anonym");
+        newUser.setPassword("123456");
+        newUser.setAge(18);
+        User userFromDb = registrationService.register(newUser);
+
+        assertNotNull(userFromDb.getId());
+        assertEquals(newUser.getLogin(), userFromDb.getLogin());
+    }
```

Comment:
You are only checking if the user's ID is not null and if the login is the same. You should also check if the password and age are the same. Additionally, you should check if the user was added to the storage.

Reply:
It isn't part of this method. It's part of others tests!